MastodonSDK/Sources/MastodonSDK/Entity/Mastodon+Entity+Account.swift

```diff
@@ -84,6 +84,64 @@ extension Mastodon.Entity {
     }
 }
 
+extension Mastodon.Entity.Account: Hashable {
+    public func hash(into hasher: inout Hasher) {
+        hasher.combine(id)
+        hasher.combine(username)
+        hasher.combine(acct)
+        hasher.combine(url)
+        hasher.combine(displayName)
+        hasher.combine(note)
+        hasher.combine(avatar)
+        hasher.combine(avatarStatic)
+        hasher.combine(header)
+        hasher.combine(headerStatic)
+        hasher.combine(locked)
+        hasher.combine(emojis)
+        hasher.combine(discoverable)
+        hasher.combine(createdAt)
+        hasher.combine(lastStatusAt)
+        hasher.combine(statusesCount)
+        hasher.combine(followersCount)
+        hasher.combine(followingCount)
+        hasher.combine(moved)
+        hasher.combine(fields)
+        hasher.combine(bot)
+        hasher.combine(source)
+        hasher.combine(suspended)
+        hasher.combine(muteExpiresAt)
+    }
+}
+
+extension Mastodon.Entity.Account: Equatable {
+    public static func == (lhs: Mastodon.Entity.Account, rhs: Mastodon.Entity.Account) -> Bool {
+        return lhs.id == rhs.id &&
```

Comment:
@kimar Stupid question: Can we compare `hashValue` here?

Reply:
Hm, I would have guessed so, but after reading https://developer.apple.com/documentation/swift/hashable/hashvalue, specifically `Hash values are not guaranteed to be equal across different executions of your program.`. I'm not so sure anymore wether we should use it...